test/unit/com/unifina/ModuleTestingSpecification.groovy

```diff
@@ -23,6 +23,7 @@ class ModuleTestingSpecification extends BeanMockingSpecification {
 	}
 
 	protected Globals mockGlobals(Map context=[:], SecUser user = new SecUser(timezone: "UTC")) {
+		user.save(failOnError: true, validate: false)
```

Comment:
Hmm probably shouldn't add this here because it requires you to pollute tests with `@Mock(SecUser)`, even if the test doesn't need SecUsers to be saved/loaded from db?

You should just `save` the user and `@Mock(SecUser)` it in the tests which load it from the db (such as your new `MessageChainUtilSpec`).